lepton/onprem.go

```diff
@@ -113,6 +113,31 @@ func (p *OnPrem) DeleteImage(ctx *Context, imagename string) error {
 	return nil
 }
 
+// SyncImage syncs image from onprem to target provider provided in Context
+func (p *OnPrem) SyncImage(config *Config, target Provider, image string) error {
+	imagePath := path.Join(localImageDir, image+".img")
```

Comment:
when running `ops image sync <image_name>`, omit `.img`
might need change, related to #517